### DISCUSSION TITLE
How to draw a square wave

### DISCUSSION THREAD

**Discussion by ricobeck** [patrykandpatrick/vico#762] [Questions]
### Question

I'm trying to create a chart for a workout with a couple of intervals. That means that I need some kind of square wave drawing. I tried to achieve this by utilizing a line chart but it seems that somehow the x coordinate is overwritten when adding a second entry.

What would be the best way to draw this?
![CleanShot 2024-06-10 at 17 14 06@2x](https://github.com/patrykandpatrick/vico/assets/155457/6e48ff75-3f87-4eaa-93c7-a872dc2355fc)

### Vico version(s)

2.0 alpha

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @ricobeck. Please see #640.
